internal/function_window.go

```diff
@@ -406,6 +406,52 @@ func (f *WINDOW_LAG) Done(agg *WindowFuncAggregatedStatus) (Value, error) {
 	return agg.Values[len(agg.Values)-f.offset-1], nil
 }
 
+type WINDOW_LOGICAL_AND struct {
+}
+
+func (f *WINDOW_LOGICAL_AND) Done(agg *WindowFuncAggregatedStatus) (Value, error) {
+	values, err := agg.RelevantValues()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cond := range values {
+		b, err := cond.ToBool()
+		if err != nil {
+			return nil, err
+		}
+
+		if !b {
+			return BoolValue(false), nil
+		}
+	}
+
+	return BoolValue(true), nil
+}
+
+type WINDOW_LOGICAL_OR struct {
+}
+
+func (f *WINDOW_LOGICAL_OR) Done(agg *WindowFuncAggregatedStatus) (Value, error) {
+	values, err := agg.RelevantValues()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cond := range values {
+		b, err := cond.ToBool()
+		if err != nil {
+			return nil, err
+		}
+
+		if b {
+			return BoolValue(true), nil
+		}
+	}
+
+	return BoolValue(true), nil
```

Comment:
Should this not be `BoolValue(false)`? Like if there are no trues we return False?


Might make sense to include a test case like this:
```
SELECT LOGICAL_OR(a) OVER (ORDER BY b), a, b
FROM (
  SELECT CAST(a AS BOOL) AS a, b FROM UNNEST([
    STRUCT(False AS a, 1 AS b), 
    STRUCT(False AS a, 2 AS b), 
    STRUCT(True AS a, 3 AS b), 
    STRUCT(False AS a, 4 AS b)
  ])
);
```
and also one that tests what happens when there are no rows:
```
SELECT LOGICAL_OR(a) OVER (ORDER BY a)
FROM (
  SELECT CAST(a AS BOOL) AS a FROM UNNEST([]) a
);
```